lib/acts_as_tree.rb

```diff
@@ -185,7 +189,7 @@ module TreeWalker
     #   <%= link_to "#{' '*level}#{page.name}", page_path(page) %><br />
     # <% end %>
     #
-    def walk_tree(_options = {}, level = 0, node = nil, &block)
+    def walk_tree(_options = {}, level = 0, node = nil, root = nil, &block)
```

Comment:
I'm not sure if it's a good idea to add an extra parameter for something this specific. How about putting it in the `_options` hash, instead?

It could also use some documentation.